cssdb.json

```diff
@@ -12,9 +12,9 @@
       "safari": "10",
       "opera": "24",
       "ios_saf": "9.3",
-      "android": "97",
+      "android": "98",
```

Comment:
Should we fix these upstream?

Seems an error that we always get latest android for these features.
This happens because there is no entry for this browser right?